FirebaseMLModelDownloader/Sources/FileDownloader.swift

```diff
@@ -68,10 +70,11 @@ class ModelFileDownloader: NSObject, FileDownloader {
   func downloadFile(with url: URL,
                     progressHandler: @escaping (Int64, Int64) -> Void,
                     completion: @escaping (Result<FileDownloaderResponse, Error>) -> Void) {
-    // TODO: Fail if download already in progress
+    // TODO: Fail if download already in progress.
     self.completion = completion
     self.progressHandler = progressHandler
     let downloadTask = downloadSession.downloadTask(with: url)
+    /// Begin or resume model download.
```

Comment:
Nit: only 2 `//` since it's not a documentation comment but a regular comment.

Reply:
Good catch. Updated everywhere not a property/method/class description to be `//`.